internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -393,11 +391,11 @@ Resources:
                   - !Sub "/${RulePath}/*"
       ListenerArn: !GetAtt EnvControllerAction.HTTPSListenerArn
       Priority: !GetAtt HTTPSRulePriorityAction.Priority
+{{- else}}  {{/* elseif .HTTPSListener */}}
```

Comment:
What do you think of creating two separate partials `http-listener` and `https-listener`?

Reply:
:+1: